String/test/LengthofLongestSubstringWithoutRepetition.test.js

```diff
@@ -12,15 +12,15 @@ describe('LengthOfLongestSubstring', () => {
     expect(lengthOfLongestSubstring('bbbbb')).toBe(1)
     expect(lengthOfLongestSubstring('pwwkew')).toBe(3)
     expect(lengthOfLongestSubstring(' ')).toBe(1)
-    expect(lengthOfLongestSubstring('abcdefghijklmnaaaaa')).toBe(13)
+    expect(lengthOfLongestSubstring('abcdefghijklmnaaaaa')).toBe(14)
```

Comment:
That's odd, how did this pass our CI previously?

Reply:
It didn't :)

See https://github.com/TheAlgorithms/JavaScript/actions/runs/6383041840/job/17322888064

Reply:
Merged without CI passing? Weird, GitHub should prevent me from doing that.

Reply:
Bruh why was it marked as passed?
<img width="849" alt="Screenshot 2023-10-03 at 12 14 12 PM" src="https://github.com/TheAlgorithms/JavaScript/assets/68542775/2c1b1868-e0e8-400f-bfbf-7775021e6811">

Reply:
I guess this is an artifact of running tests only on changed files. With my PR here, this won't happen again :)

Reply:
This will ideally also prevent causing regressions in dependents, so something like changing the quick sort pivot selection strategy breaking quick select shouldn't go unnoticed in the future :)